packages/components/text/typography.story.js

```diff
@@ -3,8 +3,8 @@ import styled from '@emotion/styled';
 import { storiesOf } from '@storybook/react';
```

Comment:
This file can be renamed to `text.story.js` and placed in the `text/src`

Reply:
Yeah, let's do that :)